packages/components/src/experience-control/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button } from '@wordpress/components';
+import clsx from 'clsx';
+import { IconBad, IconGood, IconNeutral } from './icons';
+
+import './style.scss';
+
+enum Experience {
+	GOOD = 'good',
+	NEUTRAL = 'neutral',
+	BAD = 'bad',
+}
+
+const ExperienceControlOption = ( {
+	className,
+	isSelected,
+	onClick,
+	children,
+}: {
+	className?: string;
+	isSelected: boolean;
+	onClick: () => void;
+	children: React.ReactNode;
+} ) => (
+	<Button
+		className={ clsx( 'experience-control__button', className, {
+			'is-selected': isSelected,
+		} ) }
+		onClick={ onClick }
+	>
+		<div className="experience-control__button-content">{ children }</div>
+	</Button>
+);
+
+const ExperienceControlBase = ( {
+	label,
+	children,
+	helpText,
+}: {
+	label: string;
+	children: React.ReactNode;
+	helpText?: string;
+} ) => (
+	<div className="experience-control">
+		<div className="experience-control__label">{ label }</div>
+		<div className="experience-control__buttons">{ children }</div>
+		{ helpText && <div className="experience-control__help-text">{ helpText }</div> }
+	</div>
+);
+
+export function ExperienceControl( {
```

Comment:
Can we implement this component inline in A4A? We can move it to `@automattic/components` later if we need that.

Reply:
@tyxla, since we designed it in the context of Automattic Components, wouldn't it make sense for it to be ready for use?

dhW7zYBamXJIeJIE3wy5f4-fi-111_846

Reply:
OK, I didn't realize that's the case - thanks for the nudge. Sounds fine to me in that case 👍

Reply:
Nice! Thank you!

Reply:
Thank you both! Sorry for not giving enough context. Updated the PR description